creator-node/src/fileManager.js

```diff
@@ -524,7 +500,7 @@ async function saveFileForMultihashToFS(
         data: { storageLocation }
       })
 
-      return
+      return { storagePath: storageLocation }
```

Comment:
should we also modify https://github.com/AudiusProject/audius-protocol/blob/1b267483582d85e4089830993c5a252a68bc3f52/creator-node/src/fileManager.js#L436

Reply:
? can you point me to that line on this branch - i dont see it

Reply:
i think i already changed

Reply:
oh yeah you did, sorry